app/v4/createForeignPropertyPeriodSummary/CreateForeignPropertyPeriodSummaryValidatorFactory.scala

```diff
@@ -28,6 +29,11 @@ import javax.inject.{Inject, Singleton}
 class CreateForeignPropertyPeriodSummaryValidatorFactory @Inject() (appConfig: AppConfig) {
 
   def validator(nino: String, businessId: String, taxYear: String, body: JsValue): Validator[CreateForeignPropertyPeriodSummaryRequestData] =
-    new Def1_CreateForeignPropertyPeriodSummaryValidator(nino, businessId, taxYear, body, appConfig)
+    if (taxYear >= "2024-25") {
+      new Def2_CreateValidator(nino, businessId, taxYear, body)
```

Comment:
Seems odd that these have different names, should this be Def2_CreateForeignPropertyPeriodSummaryValidator?